apollo-router/tests/snapshots/integration_tests__defer_default_variable-3.snap

```diff
@@ -3,10 +3,9 @@ source: apollo-router/tests/integration_tests.rs
 expression: first
 ---
 {
-  "data": {
```

Comment:
This file is updated because the `if` parameter is set to `false` in the defer directive.